### DISCUSSION TITLE
how to set the method solver uses in YALMIP

### DISCUSSION THREAD

**Discussion by P11GH** [yalmip/YALMIP#1256] [Problems using YALMIP] [answered]
I am using the solver gurobi in YALMIP and want set the method in the gurobi solver to concurrent (https://www.gurobi.com/documentation/9.5/refman/method.html#parameter:Method). Is the any way to do it in YALMIP?

**Comment by johanlofberg** [comment, marked as answer]
As always https://yalmip.github.io/command/sdpsettings/

**Reply by P11GH** [reply]
thank you very much
